### PR TITLE
LPS-91096 Unable to download personal data as admin

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/security/permission/resource/logic/UADExportBackgroundTaskExecutorPermissionLogic.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+public class UADExportBackgroundTaskExecutorPermissionLogic
+	implements ModelResourcePermissionLogic<BackgroundTask> {
+
+	@Override
+	public Boolean contains(
+			PermissionChecker permissionChecker, String name,
+			BackgroundTask backgroundTask, String actionId)
+		throws PortalException {
+
+		return _groupPermission.contains(
```
Comment (drewbrokke) [inline modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/security/permission/resource/logic/UADExportBackgroundTaskExecutorPermissionLogic.java]: What is the reason to use `GroupPermission` in this context?  In theory this is a company-wide sweep, so "Group" to me seems overly scoped.

Comment (stsquared99) [reply modules/apps/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/security/permission/resource/logic/UADExportBackgroundTaskExecutorPermissionLogic.java]: Is there an equivalent for company-level permissions? Also, I'm only seeing EXPORT_IMPORT_PORTLET_INFO used at the group level, so it's a bit beyond my knowledge

**Comment by drewbrokke** [comment]
Just started reviewing :)

**Comment by drewbrokke** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/72587

**Pull request closed without merging**